kubernetes/celery-beat/deployment.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: celery-beat-deployment
+  labels:
+    app: celery-beat
+spec:
+  replicas: {{.Values.celeryBeat.replicaCount}}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: celery-beat
+  template:
+    metadata:
+      labels:
+        app: celery-beat
+    spec:
+      containers:
+      - name: celery-beat
+        image: {{.Values.django.image.repo}}:{{.Values.django.image.tag}}
+        command: [ "/entrypoint", "/start-celerybeat"]
```

Comment:
[Importance 2/10]

Entrypoint doesn't work in kubernetes (as you noticed).

Maybe we can rework images to have plain script instead of two scripts
Eg just

/start-celerybeat

Reply:
Current thing is fine, don't want to bother reworking images. :)